presentation/src/main/java/com/yapp/bol/presentation/view/match/dialog/result_record/ResultRecordDialog.kt

```diff
@@ -0,0 +1,50 @@
+package com.yapp.bol.presentation.view.match.dialog.result_record
+
+import android.app.Dialog
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import com.yapp.bol.presentation.R
+import com.yapp.bol.presentation.databinding.GameResultRecordDialogBinding
+import com.yapp.bol.presentation.model.ResultRecordItem
+import com.yapp.bol.presentation.utils.dialogWidthResize
+
+class ResultRecordDialog(
+    private val context: Context,
+    private val resultRecordItem: ResultRecordItem,
+) : Dialog(context) {
+
+    private lateinit var binding: GameResultRecordDialogBinding
+
+    private val resultRecordAdapter by lazy {
+        ResultRecordAdapter()
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        binding = GameResultRecordDialogBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+        context.dialogWidthResize(this, 0.9f, true)
+        binding.rvPlayers.adapter = resultRecordAdapter
+        resultRecordAdapter.submitList(resultRecordItem.player)
```

Comment:
```suggestion
        binding = GameResultRecordDialogBinding.inflate(layoutInflater)
        setContentView(binding.root)

        window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
        context.dialogWidthResize(this, 0.9f, true)

        binding.rvPlayers.adapter = resultRecordAdapter
        resultRecordAdapter.submitList(resultRecordItem.player)
```
적절한 들여쓰기는 가독성이 좋아집니다!, 또한 추상화 레벨을 통일 시키는 것이 좋을 것 같아요